tdvt/samples/connector-tests/connectionbuilder.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version='1.0' encoding='utf-8' ?>
+<testCases>
+<!-- Driver Present -->
+<connection  installedDrivers64='PostgreSQL Unicode(x64)'  bitness='64' class='postgres_odbc' sslmode='require' username='test' password='password' dbname='TestV1' odbc-connect-string-extras='' port='5432' schema='' server='localhost' >DRIVER={PostgreSQL Unicode(x64)};DATABASE=TestV1;SERVER=localhost;UID=test;PWD=password;PORT=5432;BOOLSASCHAR=0;LFCONVERSION=0;UseDeclareFetch=1;Fetch=2048;sslmode=require</connection>
```

Comment:
Remove `schema` in both.  Not a required attribute.

Reply:
Fixed